rbac/providers/common/outbound_filters.py

```diff
@@ -43,6 +45,29 @@ def outbound_group_filter(sawtooth_group, provider):
     for key, value in GROUP_TRANSFORM.items():
         if key in sawtooth_group:
             aad_group[value[provider]] = sawtooth_group[key]
-        else:
-            aad_group[value[provider]] = None
     return aad_group
+
+
+def outbound_user_creation_filter(sawtooth_user, provider):
+    """Takes in a user dict from queue_outbound and formats it to a provider's specs.
+    :param: user > dict > a dictionary representing a user
+    :param: provider > str > inbound provider type (azure, ldap)
+    """
+    if provider != "azure" and provider != "ldap":
+        raise TypeError("Provider must be specified with a valid option.")
+    aad_user = {}
```

Comment:
Seems we are using both providers, perhaps just change this to just `user = {}`

Reply:
Ah yes will change